### PR TITLE
feat: Add common *.config.js dev configs to default exclude.

### PR DESCRIPTION
**Pull request by coreyfarrell**
Ref istanbuljs/nyc#1018

### CODE REVIEW

File: packages/test-exclude/index.js [hunk 1]
```diff
@@ -180,7 +180,7 @@ exportFunc.defaultExclude = [
     '**/*{.,-}test.js',
     '**/__tests__/**',
     '**/node_modules/**',
-    '**/babel.config.js'
+    '**/{babel,nyc}.config.js'
```
Comment (JaKXz) [inline packages/test-exclude/index.js]: you could also add `jest`

Comment (coreyfarrell) [reply packages/test-exclude/index.js]: Sure.  Going to also add `ava`.  Any others you think?

Comment (JaKXz) [reply packages/test-exclude/index.js]: Not off the top of my head

Comment (JaKXz) [reply packages/test-exclude/index.js]: `webpack`?

Comment (coreyfarrell) [reply packages/test-exclude/index.js]: Added `rollup` as well.

**Pull request merged**